opal-core-ws/src/main/java/org/obiba/opal/web/system/plugins/VCFStorePluginResource.java

```diff
@@ -0,0 +1,49 @@
+package org.obiba.opal.web.system.plugins;
+
+import org.obiba.opal.core.runtime.OpalRuntime;
+import org.obiba.opal.spi.ServicePlugin;
+import org.obiba.opal.spi.vcf.VCFStoreService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+@Component
+@Path("/system/plugins/vcf-store/{name}")
+public class VCFStorePluginResource {
+
+  public enum PLUGIN_STATUS { RUNNING, STOPPED }
+
+  @Autowired
+  private OpalRuntime opalRuntime;
+
+  @GET
+  public Response get(@PathParam("name") String name) {
+    Optional<VCFStoreService> vcfService = getVCFService(name);
+    Response.ResponseBuilder response = Response.status(vcfService.isPresent() ? Response.Status.OK : Response.Status.NOT_FOUND);
```

Comment:
This code is useless: make sure there is a mapper for the NoSuchElementException